website/docs/r/dns_record_set.markdown

```diff
@@ -10,11 +10,21 @@ description: |-
 
 Manages a set of DNS records within Google Cloud DNS.
 
-## Example Usage
+## Examples Usage
```

Comment:
I think if we're going to pluralise, it would be "Example Uses", but honestly, I'd prefer this either stay "Example Usage" or be just "Examples".